content/kubermatic/master/installation/install_kkp_CE/add_seed_cluster_CE/_index.en.md

```diff
@@ -67,37 +84,51 @@ minio:
     secretKey: "YOUR-SECRET-KEY"
 ```
 
-As a good practice, we also recommend installing the `s3-exporter` Helm chart, which provides metrics regarding user cluster backups.
+As a good practice, we also recommend installing the `s3-exporter` Helm chart, which provides metrics regarding user
+cluster backups.
 
-### Install Charts
+## Installation
 
-After configuring the required options, you can install the charts:
+The Kubermatic Installler is the recommended way to setup new seed clusters. A manual installation is possible, however
+more work needs to be done.
 
-**Helm 3**
+### Using the Installer
+
+Similar to how the master cluster can be installed with the installler, run the `deploy` command. You still need to
+manually ensure that the StorageClass you configured for Minio exists already.
 
 ```bash
-helm --namespace minio upgrade --install --wait --values /path/to/your/helm-values.yaml minio charts/minio/
-helm --namespace kube-system upgrade --install --wait --values /path/to/your/helm-values.yaml s3-exporter charts/s3-exporter/
+export KUBECONFIG=/path/to/seed-cluster/kubeconfig
+./kubermatic-installer deploy kubermatic-seed --config kubermatic.yaml --helm-values values.yaml
 ```
 
-## Add CRDs for kubermatic components in the seed cluster
+The command above will take care of installing/updating the CRDs, setting up Minio and the S3-exporter and attempts
```

Comment:
This slightly conflicts with the overview section. In the Overview section, you say that the CRDs must be installed by the user, but now you say that the KKP installer would do that. If those CRDs are different, I would clarify that here.

Reply:
> This slightly conflicts with the overview section. In the Overview section, you say that the CRDs must be installed by the user, but now you say that the KKP installer would do that. If those CRDs are different, I would clarify that here.

Wrong. The overview says "You must setup the CRDs and Helm charts (preferably using the KKP installer, but can also be done manually)".